setup.py

```diff
@@ -44,13 +48,17 @@ def read(path: str) -> str:
         ],
     },
     classifiers=[
-        "Development Status :: 3 - Alpha",
+        "Development Status :: 4 - Beta",
```

Comment:
The package is constantly used on a few projects, so I think it deserves to get the "Beta" status, at least ;], or even a higher ranking.